crates/sb_core/js/bootstrap.js

```diff
@@ -534,6 +541,9 @@ globalThis.bootstrapSBEdge = (opts, extraCtx) => {
 			'writeTextFile': true,
 			'readFile': true,
 			'readTextFile': true,
+			'mkdir': true,
+			'makeTempDir': true,
```

Comment:
hmm..how would `makeTempDir` works? Does it end up reusing the `/tmp` configured for the worker? and what if tmpFs is disabled?

Reply:
> hmm..how would makeTempDir works?
This is tied to the current context: if the path in the actual file system of tmpfs is `/tmp/.xmeow`, then the result of makeTempDir must be `/tmp/.xmeow/.xmeowmeow`.

> and what if tmpFs is disabled?
Currently, tmpfs is unconditionally set to be enabled, but if it is not, FsError::Unsupported will be returned by prefixfs.

Reply:
so I assume the `dir` option in `makeTempDir` has no effect? But users can provide `prefix` and `suffix`? https://docs.deno.com/api/deno/~/Deno.MakeTempOptions

Reply:
I'll show you how this works with a screencast shortly.

Reply:
https://github.com/user-attachments/assets/0a30514e-35ae-42be-a81e-3c5a26932b2b

cc @laktek